src/main/java/org/elasticsearch/plugin/ingest/langdetect/LangDetectProcessor.java

```diff
@@ -38,20 +39,35 @@
     private final String field;
     private final String targetField;
     private final ByteSizeValue maxLength;
+    private final boolean ignoreMissing;
 
-    public LangDetectProcessor(String tag, String field, String targetField, ByteSizeValue maxLength) throws IOException {
+    public LangDetectProcessor(String tag, String field, String targetField, ByteSizeValue maxLength, boolean ignoreMissing)
+            throws IOException {
         super(tag);
         this.field = field;
         this.targetField = targetField;
         this.maxLength = maxLength;
+        this.ignoreMissing = ignoreMissing;
+    }
+
+    boolean isIgnoreMissing() {
```

Comment:
is this used at all?

Reply:
I though I would use it for testing this component, but then I test it in different way.
I removed this section.